gobblin-core/src/main/java/gobblin/converter/csv/PercentageDoubleStringConverter.java

```diff
@@ -0,0 +1,68 @@
+package gobblin.converter.csv;
+
+import java.math.BigDecimal;
+
+import com.google.gson.JsonArray;
+import com.google.gson.JsonElement;
+import com.google.gson.JsonObject;
+
+import gobblin.configuration.WorkUnitState;
+import gobblin.converter.Converter;
+import gobblin.converter.DataConversionException;
+import gobblin.converter.SchemaConversionException;
+import gobblin.converter.SingleRecordIterable;
+
+
+/**
+ * This converter simply converts double string than ends with % to a validate double number.
+ * e.g. 99% -> 0.99
+ */
+public class PercentageDoubleStringConverter extends Converter<JsonArray, JsonArray, JsonObject, JsonObject> {
+  @Override
+  public Converter<JsonArray, JsonArray, JsonObject, JsonObject> init(WorkUnitState workUnit) {
+    super.init(workUnit);
+    return this;
+  }
+
+  @Override
+  public JsonArray convertSchema(JsonArray inputSchema, WorkUnitState workUnit)
+      throws SchemaConversionException {
+    return inputSchema;
+  }
+
+  @Override
+  public Iterable<JsonObject> convertRecord(JsonArray outputSchema, JsonObject inputRecord, WorkUnitState workUnit)
+      throws DataConversionException {
+
+    JsonObject outputRecord = new JsonObject();
+    for (int i = 0; i < outputSchema.size(); i++) {
+      JsonElement schemaElement = outputSchema.get(i);
+      String col = getColumnName(schemaElement);
+      JsonElement record = inputRecord.get(col);
+
+      if ("double".equals(getTypeName(schemaElement))) {
+        String recordString = record.getAsString();
+        if (recordString.endsWith("%")) {
```

Comment:
So far, the code never breaks, and I hope that the Google's data is consistent. But I've added a trim() to handle your case. We might only change the code as needed because there could be a lot of variations.